types/authorization.yaml

```diff
@@ -25,6 +25,14 @@ types:
       - NoCertificateAvailable
       - CertChainError
       - ContractCancelled
+  TokenValidationStatus:
+    description: Ongoing token validation status
+    type: string
+    enum:
+      - Idle
+      - Processing
+      - Accepted
```

Comment:
Should we have different enums for accepted and used to stop and accepted and used to start a session?

Reply:
What's the difference? Token is validated and accepted, right?

Reply:
Depending on what the use case is I think it does make a difference if the token is used to stop or  start a session

Reply:
I disagree. The following event will come later on. I think they added those return values just to make the code testable

Reply:
I don't understand - what is the following event ?

Reply:
SessionEvent -> TransactionStarted/TransactionStopped

Reply:
TransactionStarted: Signaled when a transaction has been started. Transaction starts at the point that all conditions for charging are met: EV is connected and user has been authorized
TransactionFinished: Signaled when the transaction finished. Transaction finishes at the point where one of the preconditions for charging irrevocably becomes false: When a user swipes to stop the transaction and the stop is authorized.

Reply:
My thinking was that we might want to differentiate if the token was used to stop (e.x show a yellow check mark) or to start (e.x Show a Green Check Mark)

Do you suggest to use the session event for this?

Reply:
As discussed offline, we can always extend this message later
I'm creating a PR